api/src/lib/aws.ts

```diff
@@ -114,6 +116,20 @@ export async function s3PutSignedUrl(
   return url
 }
 
+export async function deleteUploadFile(upload: Upload) {
+  const fileKey = `uploads/${upload.agency.organizationId}/${upload.agencyId}/${upload.reportingPeriodId}/${upload.id}/${upload.filename}`
```

Comment:
Any necessity for a simple try / catch here?

Reply:
good point, will add as a follow-on PR!